lib/hooks.js

```diff
@@ -22,55 +20,19 @@ function Hookable() {
  * List of hooks available
  */
 Hookable.afterInitialize = null;
```

Comment:
Can we take one step further in this cleanup, remove entire `lib/hooks.js` and then re-implement triggering `afterInitialize` directly in `lib/model.js`?